### DISCUSSION TITLE
[Working Now]Importing FlyteRemote doesn't seem to work

### DISCUSSION THREAD

**Discussion by TheCthulhuKid** [flyteorg/flyte#1326] [Q&A] [answered]
This question can be deleted but in the interim here is what fixed it.

After upgrading it seemed that I could not import `FlyteRemote`. After setting the interpreter in PyCharm the skeletons were rebuilt automatically but I was unable to access it. After rebuilding the skeletons manually it worked.

Silly mistake on my part and relying on the IDE a wee bit too much.

**Comment by georgesnelling** [comment, marked as answer]
Thanks for docing the resolution Gilead.

**Reply by TheCthulhuKid** [reply]
My pleasure. If it can help anyone else that stumbles across it I m happy =)
